validator/client/src/test/java/tech/pegasys/teku/validator/client/loader/ValidatorLoaderTest.java

```diff
@@ -151,6 +160,9 @@ void initializeValidatorsWithBothLocalAndExternalSigners(@TempDir Path tempDir)
                         + File.pathSeparator
                         + tempDir.toAbsolutePath().toString()))
             .build();
+    when(httpClient.send(any(), ArgumentMatchers.<HttpResponse.BodyHandler<Void>>any()))
+        .thenReturn(upcheckResponse);
+
```

Comment:
these look like they're added in just to satisfy the test? maybe just put them in a `@BeforeEach` if they're not important for the test?
If they're expected, we might want to verify they occur...

Reply:
fixed by upcheck PR